postgres/src/pool.cpp

```diff
@@ -28,9 +31,15 @@ pool_t::~pool_t() {
 
 void
 pool_t::execute(std::function<void(pqxx::connection_base&)> function) {
```

Comment:
Are the calling this method code REALLY ready for catching newly introduced exception?